andela/pippin-ah-frontend
# [Feature: #164217377]Add created article to redux store

#### What does this PR do?
- It ensures newly created articles are added to the redux store

#### Description of Task to be completed?
- [x] Dispatch an action to add newly created articles to store

#### How should this be manually tested?
- goto `/create-article` and create an article
- navigate to the category of the newly created article and it should appear without api call if there were existing articles in store before it was created



#### What are the relevant pivotal tracker stories?
- [#164217377](https://www.pivotaltracker.com/story/show/164217377)
